src/analyzer/main.cpp

```diff
@@ -202,6 +204,16 @@ namespace
             {
                 options.concise_logging = true;
             }
+            else if (arg == "-r")
+            {
+                // Make sure there is an argument following "-r"
```

Comment:
we can do it like this for now. but as part of #53 i want to introduce a root-fs concept similar to qiling that will probably change how the registry is loaded and also everything else